libr/bin/format/pe/pe.c

```diff
@@ -854,8 +854,8 @@ const char* PE_(bin_pe_compute_authentihash)(struct PE_(r_bin_pe_obj_t)* bin) {
 
 	const char *hashtype = bin->spcinfo->messageDigest.digestAlgorithm.algorithm->string;
 	ut64 algobit = r_hash_name_to_bits (hashtype);
```

Comment:
i think you must pass the filtered string in here, like this: r_str_replace_char (hashtype, '-', 0); (hashtype should be char* and strdupped from that .string